src/openforms/logging/service.py

```diff
@@ -75,3 +90,60 @@ def collect_failed_registrations(
         }
 
     return list(failed_registrations.values())
+
+
+def get_filtered_logs_admin_url(submission_ids: list[str], content_type: int) -> str:
+    query_params = {
+        "content_type": content_type,
+        "object_id__in": ",".join(submission_ids),
+        "extra_data__log_event__in": "prefill_retrieve_empty,prefill_retrieve_failure",
+    }
+    submissions_admin_url = furl(reverse("admin:logging_timelinelogproxy_changelist"))
+    return submissions_admin_url.add(query_params).url
+
+
+def collect_failed_prefill_plugins(since: datetime) -> list[dict[str, FailedPrefill]]:
+    logs = TimelineLogProxy.objects.filter(
+        timestamp__gt=since,
+        extra_data__log_event__in=[
+            "prefill_retrieve_empty",
+            "prefill_retrieve_failure",
+        ],
+    )
+
+    if not logs:
+        return []
```

Comment:
This will be inconsistent with the other return value type, I think it can be removed if the rest of the code safely returns an empty dict.